spec/Section 7 -- Response.md

```diff
@@ -220,6 +237,30 @@ still discouraged.
 ```
 
 
+## Path
+
+A `path` field allows for the association to a particular field in a GraphQL 
+result. This field should be a list of path segments starting at the root of the
+response and ending with the field to be associated with. Path segments
+that represent fields should be strings, and path segments that
+represent list indices should be 0-indexed integers. If the path is associated to
+an aliased field, the path should use the aliased name, since it represents a path in the response, not in the query.
+
+When the `path` field is present on a GraphQL response, it indicates that the 
```

Comment:
On subsequent results when we need to patch fields on the root can path be omitted or should it be and empty array?

Reply:
empty array

Reply:
We for now support both in BCP.